stories/module-analytics-settings.stories.js

```diff
@@ -101,6 +104,61 @@ function Settings( props ) {
 	);
 }
 
+function makeGtmPropertyStory( { permission, useExistingTag = false } ) {
+	return () => {
+		const setupRegistry = ( registry ) => {
+			const data = {
+				accountID: '152925174',
+				webPropertyID: 'UA-152925174-1',
+				ampPropertyID: 'UA-152925174-1',
+			};
+
+			const { accounts, properties, profiles } = fixtures.accountsPropertiesProfiles;
+
+			registry.dispatch( CORE_MODULES ).receiveGetModules( [
+				{
+					slug: 'tagmanager',
+					name: 'Tag Manager',
+					description: 'Tag Manager creates an easy to manage way to create tags on your site without updating code.',
+					homepage: 'https://tagmanager.google.com/',
+					internal: false,
+					active: true,
+					connected: true,
+					dependencies: [ 'analytics' ],
+					dependants: [],
+					order: 10,
+				},
+			] );
+
+			registry.dispatch( CORE_SITE ).receiveSiteInfo( {
+				homeURL: 'https://example.com/',
+				ampMode: AMP_MODE_SECONDARY,
+			} );
+
+			registry.dispatch( STORE_NAME ).receiveGetSettings( {} );
+			registry.dispatch( STORE_NAME ).receiveGetAccounts( accounts );
+			registry.dispatch( STORE_NAME ).receiveGetProperties( properties, { accountID: properties[ 0 ].accountId } );
+			registry.dispatch( STORE_NAME ).receiveGetProfiles( profiles, {
+				accountID: properties[ 0 ].accountId,
+				propertyID: profiles[ 0 ].webPropertyId,
+			} );
+
+			if ( useExistingTag ) {
+				registry.dispatch( STORE_NAME ).receiveGetExistingTag( data.webPropertyID );
+			}
+
+			registry.dispatch( STORE_NAME ).receiveGetTagPermission( {
```

Comment:
Tag permission should only be received if there is a valid existing tag.

Reply:
This tag permission is received for GTM tag itself, not for an existing tag defined in the Analytics module.